pkgs/by-name/op/open-vm-tools/package.nix

```diff
@@ -136,6 +136,10 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   postInstall = ''
+    # Create symblink for libpam.so.0
+    mkdir -p $out/lib
+    ln -s ${pam}/lib/libpam.so.0 $out/lib/libpam.so.0
+
```

Comment:
This doesn't seem right to me. Can we add pam to the path of the service instead?

Reply:
That is where app is trying to find lib.